sig-contributor-experience/charter.md

```diff
@@ -1,58 +1,93 @@
+
+- Establish policies, standards, and procedures:
+  - routine GitHub management tasks, including but not limited to: org membership, org permissions, repo creation/administration.
+  - "Org Owner" GitHub permissions, and grant/limit these privileges accordingly
+  - bot accounts, service accounts, webooks, and third-party integrations for all communication platforms that we support but most importantly, GitHub.
+- Establishing a "GitHub Administration team" that will oversee the execution of GitHub management tasks: inviting new members to the org, creating repos, executing moderation decisions, auditing permissions.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required
+- Deliver and iterate on Thursday community meeting by delivering a variety of content for a weekly overview into the contributor community.
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels.
+- Create best practices, trainings, policies, and moderation of Kubernetes communication platforms. This includes user guidelines when the contributor is the user of the platform. Moderation is an always on, never off service that we supply with trusted contributors spanning time zones. This includes immediate action when dealing with code of conduct issues in a public space that are defined below:
+    - All mailing lists/googlegroups where we have ownership, including k-dev@. At least one Owner of the mailing list must be a communications subproject OWNER. [link to mailing list section of moderation guidelines]
+    - Zoom via CNCF funding [link to zoom guidelines]
+    - YouTube via the /kubernetescommunity channel [link]
+    - Slack via CNCF funding [link]
+    - discuss.kubernetes.io via CNCF funding [link]
+- Help onboard new contributors and current into the culture, workflow, and CI of our contributor experience through the constant addition, modification and retirement of contributor documentation and programs tailored to onboarding.
+- Perform issue triage and maintain the kubernetes/community repository.  
+- Help SIGs with being as transparent and open as possible through creating best practices, guidelines, and general administration of YouTube, Zoom, and our mailing lists where applicable
+- Assist SIGs/WGs with organizational management operations as laid out in the sig governance doc [link]
+- Plan and execute on face to face Contributor Summit(s) and our own SIG Face to Face meetings as well as others (upon request).
+- Own and execute on other events that are targeted to the Kubernetes contributor such as the Steering Committee Election process.
+- Distribute contributor related news on various Kubernetes channels and liaison to CNCF for posting blogs and social media on their channels.
+- Work with CNCF to improve DevStats and increase adoption within our community to establish metrics that can be used in process improvements, mapping trends, or charting successes.
```

Comment:
This seems like over-specifying on devstats.  Maybe rephrase in terms of project health? e.g.

"Establish and share metrics that can be used to gauge project health, community health, and general trends, including:
- ongoing work with the CNCF to improve devstats
-